controller/iteration_test.go

```diff
@@ -454,11 +455,14 @@ func getChildIterationPayload(name *string) *app.CreateChildIterationPayload {
 	}
 }
 
-func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space.Space, area.Area, iteration.Iteration, iteration.Iteration) {
+// following helper function creates a space , root area, root iteration for that space.
+// Also creates a new iteration and new area in the same space
+func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space.Space, area.Area, iteration.Iteration, area.Area, iteration.Iteration) {
 	var spaceObj space.Space
```

Comment:
Comment: in golang, you can group variables
var (
otherIterationObj iteration.Iteration
 rootAreaObj area.Area
 rootIterationObj iteration.Iteration
otherAreaObj area.Area
)